README.md

```diff
@@ -101,7 +102,7 @@ Here is an example how to deploy and test the module on a single cluster.
 
 ### Before you begin
 
-Install Fybrik using the [Quick Start](https://fybrik.io/v0.5/get-started/quickstart/) guide. This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
+Install Fybrik using the [Quick Start](https://fybrik.io/dev/get-started/quickstart/) guide. This sample assumes the use of the built-in catalog, Open Policy Agent (OPA) and flight module.
```

Comment:
Do we need the `dev` version of the quick start or the `v0.6`?

Reply:
I think dev. The user is referred to the version compatibility matrix to choose the appropriate version of Fybrik....